lib/utils.sh

```diff
@@ -206,3 +206,29 @@ get_asdf_config_value() {
         get_asdf_config_value_from_file $default_config_path $key
     fi
 }
+
+asdf_repository_url() {
+  echo "https://github.com/doughsay/asdf-plugins.git"
+}
+
+initialize_or_update_repository() {
+  local repository_url=$(asdf_repository_url)
+  local repository_path=$(asdf_dir)/repository
+
+  if [ -d $repository_path ]; then
+    echo "updating plugin repository..."
+    (cd $repository_path && git pull)
```

Comment:
For these this of update I think `git fetch && git reset --hard origin/master` is more predictable than `git pull`.